packages/next/src/cli/next-info.ts

```diff
@@ -42,34 +96,28 @@ function getBinaryVersion(binaryName: string) {
   }
 }
 
-const nextInfo: CliCommand = async (argv) => {
-  const validArgs: arg.Spec = {
-    // Types
-    '--help': Boolean,
-    // Aliases
-    '-h': '--help',
-  }
-  const args = getValidatedArgs(validArgs, argv)
+function printHelp() {
+  console.log(
+    `
+    Description
+      Prints relevant details about the current system which can be used to report Next.js bugs
 
-  if (args['--help']) {
-    console.log(
-      `
-      Description
-        Prints relevant details about the current system which can be used to report Next.js bugs
+    Usage
+      $ next info
 
-      Usage
-        $ next info
+    Options
+      --help, -h    Displays this message
+      --run-diags   Runs additional diagnostics to collect more information
+      --out-file    Writes output into file instead of printing to stdout (--run-diags only)
```

Comment:
This arg seems useless since you can already pipe stdout to a file with fewer keystrokes

```
next info > file.txt
```

Reply:
That'll includes other outputs we emit to stdout, also --out-file is explcit to suppress stdout since output can be quite lengthy.